lib/govuk_publishing_components/presenters/contextual_navigation.rb

```diff
@@ -277,42 +149,46 @@ def parent_taxon_include?(taxon, taxon_list)
         end
       end
 
-      def brexit_cta_taxon_exception_list
-        # Entering and staying in the UK > Immigration offences
-        # Entering and staying in the UK > Inspections of border, immigration and asylum services
-        # Entering and staying in the UK > Refugees, asylum and human rights
-        # Transport > Local transport
-        # Education > Further and higher education > Education in prisons
+      def show_brexit_cta?
+        brexit_cta_taxon_allow_list?
+      end
+
+      def show_covid_booster_cta?
+        !covid_booster_cta_document_type_exempt? &&
+          !covid_booster_cta_taxon_exception_list?
+      end
+
+      def covid_booster_cta_document_type_exempt?
+        covid_booster_cta_document_type_exceptions.include?(content_item["document_type"])
+      end
+
+      def covid_booster_cta_document_type_exceptions
         %w[
-          fa13521f-9285-45b0-bd65-4a472a8037e7
-          0fab9131-f877-4286-b4d8-922fbfb402b6
-          08a8a69f-2825-4fe2-a4cf-c83458a5629e
-          3b4d6319-fcef-4637-b35a-e3df76321894
-          6426d1c5-93c8-4659-85d5-1f0d3368a124
+          simple_smart_answer
+          smart_answer
+          step_by_step_nav
         ]
       end
 
-      def brexit_cta_taxon_exception_list?
+      def covid_booster_cta_taxon_exception_list?
         taxons = content_item.dig("links", "taxons").to_a
         taxons.each do |taxon|
-          if brexit_cta_taxon_exception_list.include?(taxon["content_id"]) || parent_taxon_include?(taxon, brexit_cta_taxon_exception_list)
+          if covid_booster_cta_taxon_exception_list.include?(taxon["content_id"]) || parent_taxon_include?(taxon, covid_booster_cta_taxon_exception_list)
             return true
           end
         end
         false
       end
 
-      def brexit_cta_exception?
-        brexit_cta_document_exception? ||
-          brexit_cta_document_type_exception? ||
-          brexit_cta_taxon_exception_list?
-      end
-
-      def show_brexit_cta?
-        brexit_cta_taxon_allow_list? &&
-          brexit_cta_lang_allow_list? &&
-          step_by_step_count.zero? &&
-          !brexit_cta_exception?
+      def covid_booster_cta_taxon_exception_list
+        # Welfare > Death and benefits
+        # Life circumstances > Death and bereavement
+        # Life circumstances > Death registration disclosure
+        %w[
+          ac7b8472-5d09-4679-9551-87847b0ac827
+          0fffa994-b76d-4539-8bf9-2a6c6751580d
+          de978199-11e8-49b9-965e-4fa6a7ae79a8
+        ]
       end
 
       def step_by_step_count
```

Comment:
This method is no longer used. I am a bit confused about what it did at first... was it to only allow the Brexit cta on the first page of a step by step @sihugh? It looks like excluding via the `step_by_step_nav` document type should work for this, but I am wondering if there was a more complex rending reason this `step_by_step_count` method was used for Brexit? or maybe it was just only wanted on the first page?

Reply:
Good spot! I'll remove the method and fix-up. Thank you.